portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +26,35 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private ArrayList<String> canadianCities;
```

Comment:
It is generally better to declare variables as interfaces, so "List<String>". Declaring as specific implementation can get you into trouble later.

Reply:
Sounds good! I've declared it as a type List instead.